manager.py

```diff
@@ -74,7 +85,7 @@ def docker_checker():
 
     # touch a file to signal we're healthy, then consume events
     print('listening for events...', file=stderr)
-    open('/manager-ready', 'a').close()
+    open('/healthcheck/manager-ready', 'a').close()
```

Comment:
Minor: We have 3 different places for this path. It might be good to read this from a variable.